### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by yaroslav-furhala**
- [DEMO LINK](https://yaroslav-furhala.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://yaroslav-furhala.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Demo link is not working

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="container">
+        <nav class="container_1">
+          <img src="https://github.com/yaroslav-furhala/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+          class="link_box"
+          alt="moyo">
```
Comment (SanyaBratashchuk) [inline src/index.html:17]:
```suggestion
          <img 
            src="https://github.com/yaroslav-furhala/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
            class="link_box"
            alt="moyo"
          >
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Check mockup once more, and fix your structure there should be smth like that `<header><nav><ul><li><a></a></li></ul></nav></header>`

File: src/index.html (hunk 1 above)
Comment (ivanchuk-yurii) [reply src/index.html:17]: also change structure as in previous comment

File: src/style.css [hunk 2]
```diff
@@ -1 +1,53 @@
...
+  padding-left: 20px;
+  height: 20px;
+  width: 20px;
+}
+a:link {
+  line-height: 30px;
+  margin: 0;
+}
+a:visited {
+  color: #9F510B;
```
Comment (ivanchuk-yurii) [inline src/style.css:45]: move colors into variables, hex should be in lower case

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
compare your layout with design, check tests. if you have questions then ask in slack

**Pull request closed without merging**